store/filestore/file_cache.go

```diff
@@ -0,0 +1,52 @@
+package filestore
+
+import (
+	"sync"
+
+	"github.com/smancke/guble/store"
+)
+
+type cache struct {
+	entries []*cacheEntry
+	sync.RWMutex
+}
+
+func newCache() *cache {
```

Comment:
newCache is private, the other funcs with cache receiver are public/exported => can they also be private ?